cmd/influx/write.go

```diff
@@ -74,41 +90,102 @@ func cmdWrite(f *globalFlags, opt genericCLIOpts) *cobra.Command {
 	return cmd
 }
 
+func (writeFlags *writeFlagsType) dump(args []string) {
```

Comment:
It helps with the troubleshooting of CSV -> LP conversion. It is easier to find problems in user input when you dump the actual input arguments together with their default values. This is printed only when in --debug mode, together with metadata about columns that are used to extract CSV data to create protocol lines.  

Sure I can remove it, but it will be then harder to resolve user input errors. Opinionated herein, I do insist on having it here.